### PR TITLE
Hotfix wizardlet data getting reset

### PR DESCRIPTION
**Pull request by lchanmann**
## Ref
- https://nanopay.atlassian.net/browse/NP-7603
- https://nanopay.atlassian.net/browse/NP-6448

Credited to @KernelDeimos

### CODE REVIEW

File: src/foam/nanos/crunch/ui/UserCapabilityJunctionWAO.js [hunk 1]
```diff
@@ -87,6 +87,18 @@ foam.CLASS({
...
+      if ( foam.nanos.crunch.MinMaxCapabilityData.isInstance(wData) ) {
+        wData = wData.clone();
+        ['selectedData', 'selectedDataStable'].forEach(k => {
+          wData[k] = wData[k].map(v => {
+            if ( typeof v === 'string' ) return v;
+            return v?.id;
+          })
+        })
+      }
+
```
Comment (lchanmann) [inline src/foam/nanos/crunch/ui/UserCapabilityJunctionWAO.js]: Another possible fix for minmax wizardlet https://github.com/kgrgreer/foam3/pull/1943/files.

Comment (tharmaman) [reply src/foam/nanos/crunch/ui/UserCapabilityJunctionWAO.js]: So https://github.com/kgrgreer/foam3/pull/1943 is able to replace this change?

Comment (lchanmann) [reply src/foam/nanos/crunch/ui/UserCapabilityJunctionWAO.js]: Yes

Comment (lchanmann) [inline src/foam/nanos/crunch/ui/UserCapabilityJunctionWAO.js]:
```suggestion
```

**Pull request merged**